test/src/main/java/org/axonframework/test/fixture/AxonTestFixture.java

```diff
@@ -113,7 +113,7 @@ public Customization() {
         /**
          * Registers the given {@code fieldFilter}, which is used to define which Fields are used when comparing
          * objects. The {@link org.axonframework.test.fixture.AxonTestPhase.Then.Event#events} and
-         * {@link org.axonframework.test.fixture.AxonTestPhase.Then.Command#resultMessage}, for example, use this
+         * {@link org.axonframework.test.fixture.AxonTestPhase.Then.Command#resultMessageSatisfies}, for example, use this
```

Comment:
This is not true right? The filter is not used by the `satifies`, but by the `matches`